Client/Frontend/Browser/HomePanel/BraveShieldStatsView.swift

```diff
@@ -167,3 +171,13 @@ class StatView: UIView {
         fatalError("init(coder:) has not been implemented")
     }
 }
+
+
+extension BinaryInteger {
+    func toLocaleString() -> String? {
```

Comment:
I'm thinking of better name for this, maybe `decimalFormattedString`?
And change it to computed property `var decimalFormattedString: String? { code }`